src/views/Home.vue

```diff
@@ -1,22 +1,22 @@
 <template >
   <div :class="this.$store.state.css">
     <div class="home row">
-      <div v-if="this.$store.state.storeLoaded" class="row">
-        <div class="col-sm-8 offset-sm-2 col-md-6 offset-md-3 col-xl-5 offset-xl-2">
+      <div v-if="this.$store.state.storeLoaded">
+        <div class="widgetsAtHome">
```

Comment:
What do you include in widgetsAtHome ? could you clarify how you choose that name ?

Reply:
There are three different elements or widgets in page Home, this div gather all these elements together for CSS class. Indeed an other name more appropriate and more accurate can be used to prevent lack of understanding : componentsInHomePage ?

Reply:
HomePageComponents ?